docs/search/search_operators.rst

```diff
@@ -3,4 +3,252 @@
 Search
```

Comment:
```suggestion
Searching Mautic
```

You need a more specific heading here - RSTCheck is throwing an error. Please check the heading underline length if you use this suggestion :)